platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MarkerContainer.java

```diff
@@ -63,26 +63,13 @@ public Marker addBy(@NonNull BaseMarkerOptions markerOptions, @NonNull MapboxMap
       }
 
       if (markers.size() > 0) {
-        long[] ids = null;
-        if (nativeMapView != null) {
-          ids = nativeMapView.addMarkers(markers);
+        long[] ids = nativeMapView.addMarkers(markers);
```

Comment:
Looks like the `if (nativeMapView != null)` check was lost during the refactor. Wouldn't we want to reintroduce it?